assets/scss/modules/_tooltip.scss

```diff
@@ -1 +1,120 @@
+/// Tooltips Component
+/// @example markup - HTML structure
+/// <p class="dp-tooltip-right">dooplertip <span class="tooltiptext">Tooltip text</span></p>
+/// In order to specify the orientation of the tooltip you can use the: dp-tooltip-top | dp-tooltip-right | dp-tooltip-bottom | dp-tooltip-left classes if you need it.
 
+
+@mixin dp-tooltip ($bg-tooltip, $font-size-tooltip, $font-color-tooltip){
+  position: relative;
+  display: inline-block;
+  padding: 3px $dp-spaces--lv1;
+  margin: 0;
+  background: $bg-tooltip;
+  border-radius: 3px;
```

Comment:
maybe this value repeat in more places, can add in a variable?

Reply:
I can add the variable